lib/terrafying/components/letsencrypt.rb

```diff
@@ -312,6 +323,11 @@ def renew(name, bucket, domains, options={})
             policy_arn: "${aws_iam_policy.#{@name}_lambda_s3.arn}"
             }
 
+        resource :aws_cloudwatch_log_group, "#{@name}", {
```

Comment:
Do you need this or does one get automatically created by lambda?

Reply:
not sure if the lambda creates the log group (I guess it would/should be able to with the `CreateLogGroup` IAM permission), but I am following the tf example here:
https://www.terraform.io/docs/providers/aws/r/lambda_function.html

Reply:
ah it's right in the comment :D
```# This is to optionally manage the CloudWatch Log Group for the Lambda Function.
# If skipping this resource configuration, also add "logs:CreateLogGroup" to the IAM policy below.```

Reply:
I'll drop it, easier for the lambda to just create the log group

Reply:
`# This is to optionally manage the CloudWatch Log Group for the Lambda Function.
# If skipping this resource configuration, also add "logs:CreateLogGroup" to the IAM policy below.`